webpack/assets/javascripts/react_app/components/CPUCoresInput/CPUCoresInput.stories.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import CPUCoresInput from './CPUCoresInput';
+
+export default {
+  title: 'Components|Form/CPUCoresInput',
+  component: CPUCoresInput,
+  parameters: {
+    centered: { disable: true },
+  },
+};
+
+export const UseCpuOrCoresInput = () => <CPUCoresInput label="CPUs" />;
```

Comment:
It would be nice to use the [`knobs-addon`](https://github.com/theforeman/foreman-js/tree/master/packages/stories#storybookaddon-knobs) to demo the input props and the [`actions-addon`](https://github.com/theforeman/foreman-js/tree/master/packages/stories#storybookaddon-actions) to demo the `onChange` output prop.